frontend/src/app/login-form/login-form.component.ts

```diff
@@ -4,25 +4,30 @@ import { HttpService, LoginResponse } from './../services/http.service';
 @Component({
   selector: 'app-login-form',
   templateUrl: './login-form.component.html',
-  styleUrls: ['./login-form.component.scss']
+  styleUrls: ['./login-form.component.scss'],
 })
 export class LoginFormComponent {
   hide = true;
   token: string = null;
+  error = null;
```

Comment:
Consider introducing a type

Reply:
Also: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
good idea!